sonar-orchestrator-junit5/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.sonarsource.orchestrator</groupId>
+    <artifactId>orchestrator-parent</artifactId>
+    <version>3.43-SNAPSHOT</version>
+  </parent>
+  <artifactId>sonar-orchestrator-junit5</artifactId>
+  <name>Orchestrator - JUnit 5</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <version>5.9.2</version>
```

Comment:
In fact the JUnit 4 property is shared in the entire project because JUnit 4 is both used for unit tests, and as a main dependency of the "junit4" module. I think it is safer to use the same version everywhere.
But JUnit 5 is current ly only used in the "junit5" module, so I tend to prefer having the version in this module only.